src/plone/volto/profiles.zcml

```diff
@@ -2,6 +2,7 @@
     xmlns="http://namespaces.zope.org/zope"
     xmlns:genericsetup="http://namespaces.zope.org/genericsetup"
     xmlns:i18n="http://namespaces.zope.org/i18n"
+    xmlns:zcml="http://namespaces.zope.org/i18n"
```

Comment:
This works???  It should be `/zcml` at the end of this line.

Fixed directly on main in commit 8358b85b2f3e46bb8d603a78d2e77be049628d66.